@NullMDR I can't get that command to work. Tried deleting node modules and reinstalling.

```
c:\src\repos\azure-rest-api-specs>npm run prettier -- --write "specification/appconfiguration/**/*.json"

> azure-rest-api-specs-tests@0.1.0 prettier c:\src\repos\azure-rest-api-specs
> prettier "--write" "specification/appconfiguration/**/*.json"

[error] Unable to expand glob patterns: specification/appconfiguration/**/*.json !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
[error] Cannot find module 'c:\src\repos\azure-rest-api-specs\scripts\prettier-swagger-plugin' from 'c:\src\repos\azure-rest-api-specs'

npm ERR! Windows_NT 10.0.18363
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\jimmyca\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "prettier" "--" "--write" "specification/appconfiguration/**/*.json"
npm ERR! node v12.16.2
npm ERR! npm  v2.15.11
npm ERR! code ELIFECYCLE
npm ERR! azure-rest-api-specs-tests@0.1.0 prettier: `prettier "--write" "specification/appconfiguration/**/*.json"`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the azure-rest-api-specs-tests@0.1.0 prettier script 'prettier "--write" "specification/appconfiguration/**/*.json"'.
npm ERR! This is most likely a problem with the azure-rest-api-specs-tests package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     prettier "--write" "specification/appconfiguration/**/*.json"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs azure-rest-api-specs-tests
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls azure-rest-api-specs-tests
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\src\repos\azure-rest-api-specs\npm-debug.log
```

says it can't find "prettier-swagger-plugin". I can see that file in the scripts folder.

Doesn't repro for you?

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresCreate.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresCreate.json
@@ -28,8 +28,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {
@@ -56,8 +55,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresCreateWithIdentity.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresCreateWithIdentity.json
@@ -34,8 +34,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {
@@ -73,8 +72,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresGet.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresGet.json
@@ -19,8 +19,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresList.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresList.json
@@ -19,8 +19,7 @@
                   "identityClientId": null
                 }
               },
-              "privateEndpointConnections": [
-              ],
+              "privateEndpointConnections": [],
               "publicNetworkAccess": "Disabled"
             },
             "sku": {
@@ -48,8 +47,7 @@
                   "identityClientId": null
                 }
               },
-              "privateEndpointConnections": [
-              ],
+              "privateEndpointConnections": [],
               "publicNetworkAccess": "Disabled"
             },
             "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresListByResourceGroup.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresListByResourceGroup.json
@@ -20,8 +20,7 @@
                   "identityClientId": null
                 }
               },
-              "privateEndpointConnections": [
-              ],
+              "privateEndpointConnections": [],
               "publicNetworkAccess": "Disabled"
             },
             "sku": {
@@ -44,8 +43,7 @@
                   "identityClientId": null
                 }
               },
-              "privateEndpointConnections": [
-              ],
+              "privateEndpointConnections": [],
               "publicNetworkAccess": "Disabled"
             },
             "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresUpdate.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresUpdate.json
@@ -27,8 +27,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {
@@ -55,8 +54,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {

--- a/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresUpdateWithIdentity.json
+++ b/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/examples/ConfigurationStoresUpdateWithIdentity.json
@@ -33,8 +33,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {
@@ -72,8 +71,7 @@
               "identityClientId": null
             }
           },
-          "privateEndpointConnections": [
-          ],
+          "privateEndpointConnections": [],
           "publicNetworkAccess": "Disabled"
         },
         "sku": {

